migrations/20200827163426-upload-app-details.js

```diff
@@ -0,0 +1,99 @@
+module.exports = {
+  async up(db) {
+
+    const kendraServiceHelper = require("../generics/services/kendra");
+
+    global.migrationMsg = "Upload app details"
+    // return await db.collection('albums').updateOne({artist: 'The Beatles'}, {$set: {blacklisted: true}});
+
+    let appDetails = [
+      {
+        name: "samiksha",
+        displayName: "Samiksha",
+        appDescrption: "Get the app to discover more",
```

Comment:
@deepakr123 - Change it to description and logo only, please.

Reply:
resolved